src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -6694,6 +6872,7 @@ public abstract partial class Calendar : System.ICloneable
         public abstract System.DateTime ToDateTime(int year, int month, int day, int hour, int minute, int second, int millisecond, int era);
         public virtual int ToFourDigitYear(int year) { return default(int); }
     }
+#if FEATURE_COREFX_GLOBALIZATION
```

Comment:
But this type is visible in desktop / mono, so we want it in core and in UWP. So we need these types. 

---
In reply to: [68827364](https://github.com/dotnet/corefx/pull/9712#discussion_r68827364) [](ancestors = 68827364)